Azure/azure-cli
# [CosmosDB] `az cosmosdb update`: Support updating key vault key uri

**Description**
The change is to add the parameter to az cosmosdb update operation(for database account update), which will trigger enabling encryption in existing unencrypted cosmosdb account with data already present.

**Testing Guide**
az cosmosdb update --name "cmkselfservesud" --resource-group "subarthrg" --key-uri "[https://keyvaultcmk.vault.azure.net/keys/key1cmkstage/"](https://keyvaultcmk.vault.azure.net/keys/key1cmkstage/%22) --debug

The command would trigger an RP action which would internally trigger a control plane action for enabling cmk on exisiting accounts.


---

This checklist is used to make sure that common guidelines for a pull request are followed.
